responder.go

```diff
@@ -87,15 +77,15 @@ func Start(ctx context.Context, opts Config, action HookHandler) (func(), error)
 		http.Handle("/", c.ThenFunc(denyHandler))
 
 		if opts.EnableTLS {
-			certFile, keyFile := at.CertPaths()
-			log.Info().Str("addr", opts.TLSAddress).Msg("Listening for webhook callbacks")
-			err := http.ListenAndServeTLS(opts.TLSAddress, certFile, keyFile, nil)
+			log.Info().Int("port", opts.HTTPSPort).Msg("Listening for webhook callbacks")
+			err := certmagic.HTTPS([]string{}, http.DefaultServeMux)
```

Comment:
I should explicitly mention this in the godoc if it's not already, but you can leave the second argument `nil` just like how `http.ListenAndServeTLS()` works; it will default to `http.DefaultServeMux`: https://github.com/mholt/certmagic/blob/a3b276a1b44e1c2c3dcab752729976ea04f4839b/certmagic.go#L68-L70

But you can also leave it for explicitness' sake, doesn't matter either way.

Reply:
Also did you mean to leave the slice of domain names empty?